src/lumigo_opentelemetry/__init__.py

```diff
@@ -94,9 +98,20 @@ def init():
     from lumigo_opentelemetry.instrumentations import instrumentations  # noqa
     from lumigo_opentelemetry.instrumentations.instrumentations import framework
     from lumigo_opentelemetry.resources.detectors import get_resource
+    from lumigo_opentelemetry.libs.json_utils import DEFAULT_MAX_ENTRY_SIZE
 
     tracer_resource = get_resource(attributes={"framework": framework})
-    tracer_provider = TracerProvider(resource=tracer_resource)
+
+    span_attr_limit = int(
```

Comment:
I believe a more elegant way would be to do this (which is used in https://github.com/open-telemetry/opentelemetry-python-contrib/blob/7625b82dffaa04173d6f7cb733d354977dbe9c84/opentelemetry-distro/src/opentelemetry/distro/__init__.py#L37):

```
os.environ.setdefault(OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT, DEFAULT_MAX_ENTRY_SIZE)
```

Reply:
I think that doing `os.environ.setdefault(OTEL_SPAN_ATTRIBUTE_VALUE_LENGTH_LIMIT, DEFAULT_MAX_ENTRY_SIZE)` is problematic because it changes the environment variable and we want to make minimum changes to the environment.
But I agree that it is a code duplication so it should be extracted to a function.

Reply:
We truncate the span attributes to 2048 max in the ingestion pipeline anyhow. But either way works for me.